spec/models/affiliate_spec.rb

```diff
@@ -3,22 +3,35 @@
 RSpec.describe Affiliate do
 
   it { is_expected.to validate_uniqueness_of(:name) }
-
   it { is_expected.to validate_presence_of(:name) }
 
-  it "should maintain other as a constant" do
-    expect(Affiliate.OTHER).to eq(Affiliate.where(name: "Other").first)
+  it "maintains 'Other' as a constant" do
+    expect(Affiliate.OTHER).to eq(Affiliate.find_by(name: "Other"))
   end
 
-  it "should not allow OTHER to be destroyed" do
+  it "does not allow OTHER to be destroyed" do
     Affiliate.OTHER.destroy
     expect(Affiliate.OTHER).not_to be_destroyed
   end
 
-  it "should allow non-OTHER affiliates to be destroyed" do
+  it "allows non-OTHER affiliates to be destroyed" do
     affiliate = Affiliate.create!(name: "aff1")
     affiliate.destroy
     expect(affiliate).to be_destroyed
   end
 
+  describe "#other?" do
+    context "when it is the 'Other' affiliate" do
+      subject(:affiliate) { Affiliate.OTHER }
```

Comment:
Should we be using double colon as the formatting for this? `Affiliate::OTHER`. I don't have a strong feeling since we've been using the dot for a while.